bot/exts/gurkan/gurkan_stats.py

```diff
@@ -0,0 +1,110 @@
+from typing import Union
+
+from bot.bot import Bot
+from bot.constants import Emojis
+from bot.utils.is_gurkan import gurkan_check, gurkan_rate
+from discord import Color, Embed, Member
+from discord.ext.commands import Cog, Context, command
+
+
+RATE_DICT = {
+    range(1, 10): "pathetic",
+    range(10, 30): "not bad",
+    range(30, 60): "good",
+    range(60, 80): "cool!",
+    range(80, 95): "so Epic!!",
+    range(95, 100): "just wow, superb!",
+}
+
+
+class GurkanStats(Cog):
+    """Commands for showing stats on the Gurkan server."""
+
+    def __init__(self, bot: Bot) -> None::
```

Comment:
```suggestion
    def __init__(self, bot: Bot) -> None:
```
That was silly of me.